lib/tasks/views_schema.rake

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+namespace :db do
+  namespace :views do
+    desc 'Export the views to a schema file'
+    task dump_schema: :environment do
```

Comment:
Shouldn't this be `schema_dump` to be consistent with `schema_load`?

Reply:
Actuall, both should probably be db:views:schema:load and db:views:schema:dump to be consistent with rails db:schema:load and db:schema:dump